x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/pages/coverage_overview/filter_panel.test.tsx

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { fireEvent, render, within } from '@testing-library/react';
+import React, { useReducer } from 'react';
+
+import { TestProviders } from '../../../../common/mock';
+import { CoverageOverviewDashboardContext } from './coverage_overview_page';
+import { createCoverageOverviewDashboardReducer, initialState } from './reducer';
+import { renderHook, act } from '@testing-library/react-hooks';
+import { CoverageOverviewFiltersPanel } from './filters_panel';
+import { ruleStatusFilterDefaultOptions, ruleTypeFilterDefaultOptions } from './constants';
+
+const renderFiltersPanel = () => {
+  const { result } = renderHook(() =>
+    useReducer(createCoverageOverviewDashboardReducer(), initialState)
+  );
+
+  const [state, dispatch] = result.current;
+
+  return {
+    wrapper: render(
+      <TestProviders>
+        <CoverageOverviewDashboardContext.Provider value={{ state, dispatch }}>
+          <CoverageOverviewFiltersPanel isLoading={false} />
+        </CoverageOverviewDashboardContext.Provider>
+      </TestProviders>
+    ),
+    context: result,
+  };
+};
+
+describe('CoverageOverviewFiltersPanel', () => {
+  test('it renders all correct rule status filter options', () => {
+    const { wrapper } = renderFiltersPanel();
+
+    act(() => {
+      fireEvent.click(wrapper.getByTestId('coverageOverviewRuleStatusFilterButton'));
+    });
+
+    expect(wrapper.getByTestId('coverageOverviewFilterList')).toBeInTheDocument();
+    ruleStatusFilterDefaultOptions.forEach((option) => {
+      expect(
```

Comment:
It's an open to discussion question but I think we shouldn't test the content renders as `EuiSelectable` is responsible for rendering and similar tests should be inside EUI's source code. It'd be more reasonable to check if a request is sent with proper parameters after changing the filter.